docs/user-guides/manual-setup.md

```diff
@@ -0,0 +1,162 @@
+# Partner chains setup
+
+Partner Chains provides text based CLIs, for setting up a partner chain and starting nodes, called wizards.
+In the [chain builder](./chain-builder.md), [permissioned node owner](./permissioned.md), and [registered node owner](./registered.md) guides these wizards are used.
+
+This document explains what is performed by those wizards and how to perform such operations manually.
+
+For dependency setup please see the respective guide for your desired setup.
+
+## Generating keys
+
+`<node-executable> wizards generate-keys` creates 3 specific keys, stores them in the keystore and also creates a JSON file with public keys.
+These 3 keys are:
+* Aura, sr25519 scheme key, with "key-type" `aura`
+* GRANDPA, ed25519 scheme key, with "key-type" `gran`
+* cross-chain, ecdsa scheme key, with "key-type" `crch`
+
+For each of these key commands that are two commands invoked by wizard.
+The first one is
+```bash
+<node-executable> key generate --scheme <scheme> --output-type json
```

Comment:
So based on what I suggested above this would be
```
$ pc-node key generate --scheme <scheme> --output-type json
```
which to me looks a bit more "natural"

Reply:
But there is no such a thing like `pc-node`, so I prefer to be clear that it is something that will vary.